test/boost/transport_test.cc

```diff
@@ -26,6 +26,22 @@
 
 #include "test/lib/random_utils.hh"
 
+namespace cql3 {
+
+bool operator==(const cql3::raw_value_view& a, const cql3::raw_value_view& b) {
+    if (a.is_value()) {
+        return b.is_value() && b.with_value([&] (const FragmentedView auto& v2) {
+            return a.with_value([&] (const FragmentedView auto& v1) {
+                return equal_unsigned(v1, v2);
+            });
+        });
+    } else {
+        return a.is_null() == b.is_null();
+    }
+}
+
```

Comment:
The commit message says it's moved, but it's also rewritten. Why is that? did we lose access to some private members?

Reply:
The old version both uses private members (and we wanted to move this operator out of the class), and uses `operator*` which is removed in the next patch.